cmake/modules/hunter_config.cmake

```diff
@@ -44,9 +44,36 @@ macro(hunter_config)
 
   string(COMPARE NOTEQUAL "${_hunter_GIT_SUBMODULE}" "" _hunter_submodule_create)
   if(_hunter_submodule_create)
+    # get HUNTER_SUBMODULE_SOURCE_SUBDIR from CMAKE_ARGS
```

Comment:
in a separate file or in `hunter_config` as subfunction? (I think you mean the latter)